apps/documentation/project.json

```diff
@@ -32,13 +32,13 @@
           },
           {
             "glob": "**/*",
-            "input": ".ng-doc/documentation/assets",
+            "input": "ng-doc/documentation/assets",
             "output": "assets/ng-doc"
           },
           "apps/documentation/src/favicon.ico",
           "apps/documentation/src/assets"
         ],
-        "styles": ["libs/ngx-charts-on-fhir/src/default.scss", "node_modules/@ng-doc/app/styles/global.css", "apps/documentation/src/styles.scss"],
```

Comment:
I have removed this `libs/ngx-charts-on-fhir/src/default.scss` because the build is failing with the erorr not able to import the `material-icons` when using  `@import "material-icons/iconfont/material-icons.scss";` in default.scss . I have tried using import using the path 

> $material-icons-font-path: "material-icons/iconfont/" !default;
@import "material-icons";

for this import, the documentation app is working fine but the other apps are failing with the same error not able to import the `material-icons`.So I have used the same default.scss code in style.scss of documentation with updated   import for material-icon

Reply:
Documentation app can import charts-on-fhir theme in `styles.scss` instead of `project.json`, so it can set the font path itself without affecting other apps:

`apps/documentation/src/styles.scss`:
```
$material-icons-font-path: 'material-icons/iconfont/';
@import 'libs/ngx-charts-on-fhir/src/default.scss';
```